### PR TITLE
Update conventions.rst

### PR DESCRIPTION
**Pull request by LuizMarin**
A table with a summary for quick reference.

### CODE REVIEW

File: en/intro/conventions.rst [hunk 1]
```diff
@@ -172,6 +172,41 @@ automatically tied to the 'articles' table in the database), and renders to a
...
 
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| Example    | articles                          | articles_categories                   |                                                                        |
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| DB         | articles                          | articles_categories                   | Tables names are plural and underscored.                               |
+|            |                                   |                                       | Field/Column names with two or more words are underscored: first_name  |     
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| File       | ArticlesController.php            | ArticlesCategoriesController          |                                                                        |
```
Comment (markstory) [inline en/intro/conventions.rst]: Should ArticlesCategories have a .php here?

Comment (LuizMarin) [reply en/intro/conventions.rst]: adjusted

File: en/intro/conventions.rst [hunk 2]
```diff
@@ -172,6 +172,41 @@ automatically tied to the 'articles' table in the database), and renders to a
 file. None of these relationships have been configured by any means other than
 by creating classes and files that you'd need to create anyway.
 
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| Example    | articles                          | articles_categories                   |                                                                        |
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| DB         | articles                          | articles_categories                   | Tables names are plural and underscored.                               |
```
Comment (garas) [inline en/intro/conventions.rst]: As written in _Database Conventions_ chapter above, DB table name whose name contains multiple words should be _singular_plural_: `article_categories` and accordingly whole column.

File: en/intro/conventions.rst [hunk 3]
```diff
@@ -172,6 +172,41 @@ automatically tied to the 'articles' table in the database), and renders to a
...
+| Class      | ArticlesController                | ArticlesCategoriesController          |                                                                        |
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| Controller | ArticlesController                | ArticlesCategoriesController          | Plural, PascalCased, end in Controller                                 |
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| Table      | ArticlesTable.php                 | ArticlesCategoriesTable.php           | Table class names are plural, PascalCased and end in Table             |
++------------+-----------------------------------+---------------------------------------+------------------------------------------------------------------------+
+| Entity     | Articles.php                      | ArticlesCategories.php                | Entity class names are singular PascalCased and have no suffix         |
```
Comment (garas) [inline en/intro/conventions.rst]: Entity class names are singular: `Article` and `ArticleCategory`

**Comment by LuizMarin** [comment]
adjusted

File: en/intro/conventions.rst [hunk 4]
```diff
@@ -77,7 +77,7 @@ Database Conventions
 
 Table names corresponding to CakePHP models are plural and underscored. For
 example ``users``, ``article_categories``, and ``user_favorite_pages``
-respectively.
+respectively. Table name whose name contains multiple words should be singular_plural: article_categories and accordingly whole column.
```
Comment (garas) [inline en/intro/conventions.rst]: >  and accordingly whole column.

I wanted to say whole third column of the table you have added should be updated to use singular _articles_ in `ArticleCategories/ArticleCategory/article_categories/...`

**Comment by LuizMarin** [comment]
I not speak english....

Em sex, 31 de jan de 2020 13:32, Edgaras Janušauskas <
notifications@github.com> escreveu:

**Comment by LuizMarin** [comment]
I'm doing like blind


Em sex, 31 de jan de 2020 13:32, Edgaras Janušauskas <
notifications@github.com> escreveu:

File: en/intro/conventions.rst (hunk 4 above)
Comment (LuizMarin) [reply en/intro/conventions.rst]: I not speak english.... I'm doing like blind... Write down how the sentence should look, please.

**Pull request merged**